data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/ReceiverVerticle.java

```diff
@@ -16,30 +16,42 @@
 
 package dev.knative.eventing.kafka.broker.receiver;
 
+import dev.knative.eventing.kafka.broker.core.reconciler.impl.ResourcesReconcilerImpl;
+import dev.knative.eventing.kafka.broker.core.reconciler.impl.ResourcesReconcilerMessageHandler;
+import io.cloudevents.CloudEvent;
 import io.vertx.core.AbstractVerticle;
-import io.vertx.core.Handler;
+import io.vertx.core.CompositeFuture;
 import io.vertx.core.Promise;
+import io.vertx.core.eventbus.MessageConsumer;
 import io.vertx.core.http.HttpServer;
 import io.vertx.core.http.HttpServerOptions;
-import io.vertx.core.http.HttpServerRequest;
 import java.util.Objects;
 
-public class HttpVerticle extends AbstractVerticle {
+public class ReceiverVerticle extends AbstractVerticle {
 
+  private final String livenessPath;
+  private final String readinessPath;
   private final HttpServerOptions httpServerOptions;
-  private final Handler<HttpServerRequest> requestHandler;
+  private final RequestHandler<String, CloudEvent> requestHandler;
+
   private HttpServer server;
+  private MessageConsumer<Object> messageConsumer;
 
   /**
    * Create a new HttpVerticle.
    *
+   * @param livenessPath
+   * @param readinessPath
    * @param httpServerOptions server options.
    * @param requestHandler    request handler.
    */
-  public HttpVerticle(
+  public ReceiverVerticle(
+    final String livenessPath,
+    final String readinessPath,
```

Comment:
Why should this verticle know about probes?

Reply:
I'd prefer to keep it agnostic and remove noise in this verticle.

Reply:
Ok I have another idea to hide it, let me try it

Reply:
Ok what about now?